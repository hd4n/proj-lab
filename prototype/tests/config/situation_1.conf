#equipment, material
@items
ca c1
ba b1
am a1
nu n1

#effekt_nev id duration
@effects
pr p1 10
st st1 5

#agent_nev id effekt_id
@agents
vi vi1 st1

#kod_nev id agent_id aminocost nucleocost
@codes
co cd1 vi1 1 1
co c9

#mezo_nev id [tartalom]
@fields
em e1
wa w1 n1
em e2

#mezo_id1 mezo_id2
#ketiranyu kapcsolat
@neighbors
e1 w1
w1 e2

#id akt_mezo [item_id1;item_id2...] [material_id1;material_id2...] [effect_id1;effect_id2...]
@virologists
v1 e1 c1 a1 p1
v2 e2 b1