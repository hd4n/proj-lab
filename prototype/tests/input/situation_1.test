load situation_1.conf
interact v1
step v1 w1
nextround
interact v1 collect_material
craft v1 cd1
useagent v1 v2 vi1
nextround
steal v1 v2 equipments b1
equip v1 b1
getstats v1
getstats v2