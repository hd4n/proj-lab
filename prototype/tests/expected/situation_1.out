Load: sikeres
interact: v1 e1
step: v1 w1
nextround: sikeres
interact: v1 w1
craft: v1 cd1
useagent: v1 v2 vi1
nextround: sikeres
steal: v1 target: v2, equipments
equip: v1 b1

stats: v1
	fi_w1
	di_w1
	st_false
	res_82.3
	re_false
	ma_10
	ref_0
	neededAmino_0
	neededNucleo_0
	co_cd1
	ma_null
	ag_null
	eq_c1
	eqa_b1
	ef_p1+ib101

stats: v2
	fi_e2
	di_e2
	st_true
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_st1
